widgets/qna-advisor/src/App.ce.vue

```diff
@@ -475,6 +475,14 @@
             --rounding-lg: 0.75rem;
             --rounding-full: 9999px;
         "
+        :style="{
```

Comment:
You should combine this with the `style` attribute above it. `:style` can contain static CSS in it as well and still interpret both correctly.